drawpixels/src/drawpixels.cpp

```diff
@@ -489,6 +489,44 @@ static int draw_filled_rect(lua_State* L) {
   return 0;
 }
 
+static int draw_bezier(lua_State* L) {
+  int top = lua_gettop(L) + 4;
+
+  read_and_validate_buffer_info(L, 1);
+  int32_t x0 = luaL_checknumber(L, 2);
+  int32_t y0 = luaL_checknumber(L, 3);
+  int32_t xc = luaL_checknumber(L, 4);
+  int32_t yc = luaL_checknumber(L, 5);
+  int32_t x1 = luaL_checknumber(L, 6);
+  int32_t y1 = luaL_checknumber(L, 7);
+  uint32_t r = luaL_checknumber(L, 8);
+  uint32_t g = luaL_checknumber(L, 9);
+  uint32_t b = luaL_checknumber(L, 10);
+  uint32_t a = 0;
+  if (lua_isnumber(L, 11) == 1)
+  {
+    a = luaL_checknumber(L, 11);
+  }
+  
+  int max_dx = fmax(fmax(x0, xc), x1) - fmin(fmin(x0, xc), x1);
+  int max_dy = fmax(fmax(y0, yc), y1) - fmin(fmin(y0, yc), y1);
+  
+  int max_d = max(max_dx, max_dy) *2;
```

Comment:
When building on OSX I get: "use of undeclared identifier 'max'; did you mean 'fmax'?"